src/coreclr/jit/gentree.h

```diff
@@ -4401,7 +4401,7 @@ enum class CFGCallKind
 
 class CallArgs;
 
-enum class WellKnownArg
+enum class WellKnownArg : unsigned
```

Comment:
Why this change? (Seems fine, just curious)

Reply:
`CallArg::m_wellKnownArg` is 5 bits, and `WellKnownArg::SwiftSelf` has a value of 16 (also 5 bits). If `m_wellKnownArg` is signed, then we lose the most-significant bit, so comparisons like `m_wellKnownArg == WellKnownArg::SwiftSelf` fail.